readme.md

```diff
@@ -1,41 +1,90 @@
-I thought of replicating the user experience of a Primary 6 student taking the examination through an app where students will be quizzed on a series of practice either as a single user mode or through dual-users mode in a interactive game-like mode, if permitted multi-users realtime online.
+SYNOPSIS
 
-This is a quiz trivia style mcq or fill in the blanks game for single or double player to pit their skills, possibly even online if can be done, the material taken from primary school current syllabus 10 year series, for students to practise on.
+This game is my first project in General Assembly's Web Development Immersive Course. This is a quiz trivia style mcq, fill in the blanks, Q&A game APP for single or double players to pit their skills, possibly even online if can be done, the content materials are to be original from educators, for students to practise on using simple CSS and Javascript.
 
-Features music on click or embedded midi/mp3.
-Possibly weave some animations included.
 
-Option for One Player or Two Player Game
-Player One and Two Detail Registration
-Facebook Login  
-Login Screen
-Parent Email & Mobile
+GENERAL APP FEATURES
```

Comment:
Would be great to use heading elements to divide up your README files. For example: 

## General App Features